site/_config.yml

```diff
@@ -27,6 +27,7 @@ remote_theme: "mmistakes/jekyll-theme-basically-basic@1.4.5"
 
 lang: en-US
 title: MAKE Roanoke
+tagline: A place to make stuff
```

Comment:
If the sfl-merge to main is never completed, then a new branch will be needed to merge mr-theme into main.  Like sfl-merge is for sfl-sandbox, the new branch would be based off of main, mr-theme merged into it where all conflicts can be resolved, and then the new branch can be merged into main.

Reply:
You stated earlier, somewhere, that Tailwind and Jekyll are not compatible as they are.  Have you worked out a way to make Tailwind work with Jekyll yet?

I am against adding any framework until the the following conditions are met:

- Is compatible with Jekyll
- Is integrated with our build process both on GitHub and our local systems.
- It can be shown that the benefits outweigh the cost in both added complexity to our build environment and in the learning curve needed to use it.